systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,403 @@
+                .withAdminName(deleteAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(clusterName))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(topicsCountBelowQuota)
+                .withNamespaceName(namespaceName)
+                .withTopicOperation(AdminClientOperations.DELETE_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(namespaceName))
+                .build();
+        resourceManager.createResource(extensionContext, adminClientJob.adminStrimzi().build());
+
+        ClientUtils.waitForClientSuccess(deleteAdminName, namespaceName, 0);
+        String deletePodName = kubeClient().listPods("job-name", deleteAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(deletePodName, "Successfully removed all " + topicsCountBelowQuota);
+
+        // List topics after deletion
+        resourceManager.createResource(extensionContext, adminClientListJob.adminStrimzi().build());
+        ClientUtils.waitForClientSuccess(listAdminName, namespaceName, 0);
+        listPodName = kubeClient().listPods("job-name", listAdminName).get(0).getMetadata().getName();
+        String afterDeletePodLogs = kubeClient().logs(listPodName);
+        assertThat(afterDeletePodLogs.contains(topicNamePrefix), is(false));
+        assertThat(afterDeletePodLogs, not(containsString(topicNamePrefix)));
+
+        JobUtils.deleteJobWithWait(namespaceName, createAdminName);
+        JobUtils.deleteJobWithWait(namespaceName, listAdminName);
+        JobUtils.deleteJobWithWait(namespaceName, deleteAdminName);
+    }
+
+    String getAdminClientConfig(String namespace) {
+        final String saslJaasConfigEncrypted = ResourceManager.kubeClient().getSecret(namespace, kafkaUsername).getData().get("sasl.jaas.config");
+        final String saslJaasConfigDecrypted = new String(Base64.getDecoder().decode(saslJaasConfigEncrypted), StandardCharsets.US_ASCII);
+        return "sasl.mechanism=SCRAM-SHA-512\n" +
+                "security.protocol=" + SecurityProtocol.SASL_PLAINTEXT + "\n" +
+                "sasl.jaas.config=" + saslJaasConfigDecrypted + "\n";
+    }
+
+    void setupKafkaInNamespace(ExtensionContext extensionContext, String clusterName, String namespaceName) {
+        // Deploy kafka with ScramSHA512
+        LOGGER.info("Deploying shared kafka across all test cases in {} namespace", namespaceName);
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(clusterName, 1)
+            .editSpec()
+            .editKafka()
+            .withListeners(
+                new GenericKafkaListenerBuilder()
+                    .withName(Constants.PLAIN_LISTENER_DEFAULT_NAME)
+                    .withPort(9092)
+                    .withType(KafkaListenerType.INTERNAL)
+                    .withTls(false)
+                    .withNewKafkaListenerAuthenticationScramSha512Auth()
```

Comment:
is scram-sha needed by admin client? Wouldn't be easier to remove it?

Reply:
That's the whole point of the test. These throttling quota limits are set on KafkaUser not on topic etc, so i need to use some user. This was the only reasonable solution we figured out with @scholzj .

Reply:
Ok that makes sense.